docs/contributions.rst

```diff
@@ -63,17 +65,19 @@ Testing
 
 Add test cases, and run both existing cases and your new cases.
 
-If you add a new parser or modify existing parser, add new test cases
-to "Units". If you modify main part, add new test cases to
-"Tmain". The way to write and run test cases are written in "Testing
-ctags" of this document. Other than tmain test harness, you can
-specify VG=1 for running test cases under valgrind memory debugger.
+If you add a new parser or modify an existing parser, add new test
+cases to "Units". If you modify the core, add new test cases to
```

Comment:
I know we should call "core" part. However, using the name `core` may be conflicted with `core` file in meaing of `coredump`. So the our choice was `main`. Generally I use `main part` in discussions.

So I wonder the senten can be `If you modify the main part`. Can I ask you to reconsider this?